packages/react/src/main-nav/MainNavDialogNavList.tsx

```diff
@@ -0,0 +1,96 @@
+import type { PropsWithChildren } from 'react';
+import { Flex } from '../flex';
+import { boxPalette, useLinkComponent, mapSpacing, packs } from '../core';
+import { BaseButton } from '../button';
+import { focusStyles } from '../box';
+import { localPalette } from './localPalette';
+import { isItemLink, MainNavListItemType } from './MainNavList';
+
+export type MainNavDialogNavListProps = {
+	'aria-label': string;
+	activePath: string;
+	items?: MainNavListItemType[];
+};
+
+export function MainNavDialogNavList({
```

Comment:
does this duplicate or approximate existing code from app-layout?

Reply:
Nah this code was already here, and is a little different to what is in app-layout